internal/cluster/cluster.go

```diff
@@ -158,3 +161,24 @@ func marshalBase64(in interface{}) ([]byte, error) {
 
 	return buffer.Bytes(), nil
 }
+
+// AgentServiceAccount secret takes a service account json and formats it as a
```

Comment:
Question: do we use 'service account' naming here and elsewhere because the agent auth is based on GCP service accounts? The returned credential does not look like a service account value to me. Should we rather call it 'agent credential' or something such to avoid confusion?

Reply:
I have updated the command text to make this clearer, I hope. I haven't renamed the command as I think that the command name is still accurate.